src/test/java/components/PanelHeaderTest.java

```diff
@@ -0,0 +1,117 @@
+package components;
+
+import enums.Attribute;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.CreateAccountPage;
+import pages.LoginPage;
+import utils.WebDriverUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static colors.ColorsHex.*;
+import static page.url.PageUrl.*;
+
+public class PanelHeaderTest {
+    private WebDriver driver;
+    private PanelHeader panelHeader;
+    private List<WebElement> headerPanelList;
+
+    @BeforeClass
+    public void setUp() {
+        driver = WebDriverUtils.createWebDriver();
+        panelHeader = new PanelHeader(driver);
+        headerPanelList = panelHeader.getPanelHeaderList();
+    }
+
+    @BeforeMethod
+    public void beforeMethod() {
+        driver.get(HOMEPAGE_URL);
+    }
+
+    @Test
+    public void panelHeader_SignInLink_NavigatesToLoginPage() {
+        //when
+        LoginPage loginPage = panelHeader.openLoginPage();
```

Comment:
Instead of opening login and create pages, src attribute should be checked here and `panelHeader_CreateAccountLink_NavigatesToLoginPage()` tests.
If src attribute is checked, then `beforeMethod()` with refreshing page before each tests will be removed.
Refreshing page has no sense here, as only elements of panel header are checked.

Reply:
Basically these first two tests should be removed. As links and texts are validated in panelHeader_PageWrapperGreyBar_HasColorsAndTextCorrectlyDisplayed()